packages/dialect-react-ui/components/Chat/MessageBubble.tsx

```diff
@@ -132,7 +137,11 @@ export default function MessageBubble({
                       state === 'entered' &&
                         'dt-opacity-100 dt-scale-100 dt-transition-transform dt-duration-300'
                     )}
-                    onClick={() => onCancelMessage(deduplicationId)}
+                    onClick={() =>
+                      onCancelMessage(
+                        deduplicationId || message.length.toString()
```

Comment:
Why do we need to do this? Meaning, why also check for the string's length? In case we want to cancel a blockchain message?

If yes, I would propose to change this to some sort of hash from the array or something, wdyt?

Reply:
agreed, it was actually because of the broken types. Fixed types -> realized that this fallback is not needed